spring/boot1-autoconfigure/build.gradle

```diff
@@ -1,5 +1,18 @@
+final def SPRING_BOOT_VERSION = '1.5.22.RELEASE'
+
 evaluationDependsOn ':spring:boot-autoconfigure'
 
+dependencyManagement {
+    // Override Spring Boot versions.
+    dependencies {
+        dependencySet(group: 'org.springframework.boot', version: SPRING_BOOT_VERSION) {
```

Comment:
So we can remove the version in lines 31, 32 and 35.

Reply:
Indeed. Done!